docs/EARLY_TESTERS.md

```diff
@@ -32,6 +32,7 @@ We will ask early testers to participate at two points in the process:
 - [ ] Charity Engine (@rytiss, @tristanolive)
 - [ ] Fission (@bmann)
 - [ ] OrbitDB (@aphelionz)
+- [ ] Brave ()
```

Comment:
Hi @cypt4!

Before each stable release, we are making at least one Release Candidate release,
allowing projects that build on Kubo to provide early feedback.

Would Brave be interested in getting pinged when RC release happens, (adding you here) 
or do you prefer to only use final stable releases?